blog/2020-01-everything-github/README.md

```diff
@@ -0,0 +1,706 @@
+
+3. The app should be available at `https://<username>.github.io/<repositoryname>/` soon.
+   If it is not immediately available, you should first wait a moment.
+
+4. If we now check the results (in our case at https://angular-schule.github.io/everything-github-demo/) we will see a blank page! 😲
+    But no problem, we open the Console panel of the Chrome DevTools by pressing `Control+Shift+I` (Windows, Linux) or `Command+Option+I` (Mac).
+    We immediately see some red 404 errors for all files:
+
+    > Failed to load resource: the server responded with a status of 404 ()
+
+    Well, the reason for that is simple:
+    The application is configured as if it is running on the root path of the domain.
+    But this is not the case here!
+    Our application is located in the `/<repositoryname>/` path, so all links are broken.
+    This is of course adjustable.
+    Therefore, we will try the deployment a second time:
+
+   ```sh
+   ng deploy --base-href=/<repositoryname>/
+
+   # so in our case it must be
+   ng deploy --base-href=/everything-github-demo/
+   ```
+
+5. And now you should see our app running on Github Pages!
+    You can make sure that everything went well when you go into the **Settings** of the repo.
+    Scroll down there to the GitHub Pages section.
+    The following should appear here:
+
+    ![Screenshot](screenshot_4_settings.png)
+
+    **Hint:** You can also see the setting for a custom domain here.
+    Please do not try to set a value.
+    This will work for the first time, but the setting is not permanent, because `angular-cli-ghpages` will overwrite it again.
+
+
+
+## 5. Automating the Deployment with GitHub Actions
+
+But we don't want to stop here and trigger the deployment manually all the time.
+We want to make changes to the app and the deployment should be handled automatically for us.
+For this we want to use GitHub Actions.
+GitHub Actions is now available for all repsitories.
+So there is no need to register anymore, we can start right now! 🚀
+
+As with all CI/CD systems, we have to think about how to grant the necessary write permissions to our repo.
+
+### 5.1 About all tokens
+
+
+The easiest way to grant access to the repository is to implement this with tokens.
+Tokens can be used instead of a password for Git over HTTPS (*this is want we want to do*),
+or can be used to authenticate to the Github API (*angular-cli-ghpages is right now not using the Github API at all*).
+
+> **⚠️ Warning:** Treat tokens like passwords and keep them secret. Always use tokens as environment variables instead of hardcoding them into your code!
+
+
+At Github there are several types of tokens, which we have to carefully distinguish.
+The following two are relevant for us:
+
+* **`GH_TOKEN` / Personal access token** – As the name suggests, personal access tokens grant rights that a particular user has.
+    In order to prevent the release of too many rights, the range of features can be limited with so-called scopes.
```

Comment:
scopes maybe in italic when using it with "so called" the first time? _scopes_

Reply:
maybe like this?

In order to prevent the release of too many rights, the range of features can be limited with so-called "scopes".